clang/test/CodeGenCXX/builtins-eh-wasm.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -triple wasm32-unknown-unknown -fexceptions -fcxx-exceptions -target-feature +reference-types -target-feature +exception-handling -target-feature +multivalue -exception-model=wasm -emit-llvm -o - %s | FileCheck %s
+
+// Check if __builtin_wasm_throw and __builtin_wasm_rethrow are correctly
+// invoked when placed in try-catch.
+
+void throw_in_try(void *obj) {
+  try {
```

Comment:
do we also expect there to be an invoke when not placed in a try, or does `EmitRuntimeCallOrInvoke` handle that case automatically?

Reply:
It handles the case automatically (hence `CallOrInvoke`)